tests/broker/storage/connector.cc

```diff
@@ -19,30 +19,41 @@
 
 #include "com/centreon/broker/storage/connector.hh"
 #include <gtest/gtest.h>
-#include "com/centreon/broker/exceptions/config.hh"
+#include "com/centreon/exceptions/config.hh"
 #include "com/centreon/broker/exceptions/msg.hh"
+#include "com/centreon/exceptions/msg_fmt.hh"
```

Comment:
If things are well done, since you include config.hh, you don't need anymore msg_fmt.hh (it comes with the previous one).